db/migrate/20170314224434_create_entries.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEntries < ActiveRecord::Migration[5.0]
+  def change
+    create_table :entries do |t|
+      t.string :title
+      t.text :body
```

Comment:
Adding image in paperclip branch, not possible to create field:type column at this time.